pandas/tests/io/formats/test_format.py

```diff
@@ -3432,3 +3432,14 @@ def test_format_remove_leading_space_dataframe(input_array, expected):
     # GH: 24980
     df = pd.DataFrame(input_array).to_string(index=False)
     assert df == expected
+
+
+def test_nullable_float_to_string(float_ea_dtype):
```

Comment:
i think should move these to EA tests and try to do these generically (for all of the numberic types); i guess ok here as well (but group them together int. & float both with and w/o nulls)

Reply:
do we have int nullable tests for formatting?

Reply:
Not as far as I can tell, entirely possible I'm just missing them

Reply:
kk can you add (this PR would be great)

Reply:
Done. These tests could probably be cleaned up a bit also, this file is very big and testing lots of different things.

Reply:
oh yes, for sure. if you'd create an issue would be great.